### ISSUE TITLE
False positive: 'aria-allowed-attr' fails when using 'aria-errormessage' attribute

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#1043]
'aria-allowed-attr' should not fail for any of the 3 cases in the below example

- Link to live demo: https://codepen.io/iamrafan/pen/zLjyKM?editors=1000#

<pre><code>axe-core version: 3.0.3
axe-Coconut version: 3.4.0-alpha.2</code></pre>

**Comment by iamrafan** [comment]
Sure, since aria-errormessage is a global attribute, I think is should be allowed on all the elements when a fallback is provided. Then:
- it would pass 'aria-valid-attr' always 
- it would fail 'aria-attr-valid-value' when a fallback is not provided.

**Comment by iamrafan** [comment]
@WilcoFiers I mentioned the wrong rule name, this bug is about 'aria-allowed-attr'. I made the rule name changes in this thread.

**Comment by WilcoFiers** [comment]
Jup, I noticed it.
